api/vehicles.go

```diff
@@ -184,9 +185,13 @@ func (App *API) UpdateMessageHandler(w http.ResponseWriter, r *http.Request) {
 				log.WithError(err).Error("Could not load time zone information.")
 				continue
 			}
-			lastUpdate := update.Created.In(loc).Format("3:04:05pm")
 
-			message = fmt.Sprintf("<b>%s</b><br/>Traveling %s at<br/> %s mph as of %s", vehicle.VehicleName, CardinalDirection(&update.Heading), speed, lastUpdate)
+			// Query Routes that matches the update's RouteID
+
+			err = App.db.Routes.Find(bson.M{"id": update.Route}).Limit(1).One(&currentRoute)
```

Comment:
Ask me why I'm not handling the error here! Apparently, ```err```will never **not** be ```nil```, and Go won't let you set HTTP headers twice. If you find a better way to handle a potential error here, let me know.